ethereum/api/src/test/resources/org/hyperledger/besu/ethereum/api/jsonrpc/trace/chain-data/blocks.json

```diff
@@ -82,6 +82,14 @@
           "gasPrice": "0xEF",
           "to": "0x0020000000000000000000000000000000000000",
           "data": "0x0000000000000000000000000000000000000999"
+        },
+        {
+          "comment": "Self destruct and send contract balance to the same address provided in the to field.",
```

Comment:
Can this be moved to it's own transaction in a new block `0x1B`?  Since it adjusts a balance field as you can see the cascading change to test files is huge, impacting almost all of them.